cypress/e2e/crypto/decryption-failure.spec.ts

```diff
@@ -71,6 +72,7 @@ describe("Decryption Failure Bar", () => {
             homeserver = hs;
             cy.initTestUser(homeserver, TEST_USER)
                 .then((creds: UserCredentials) => {
+                    skipIfRustCrypto();
```

Comment:
because we have to let element initialise the settings object before we can tell if we're using rust crypto.

... though I could have it check the env var which configures rust crypto, rather than checking whether rust crypto is actually in use, I suppose.

Reply:
> Feels more like the it/describe should just be in an if-statement if they're to be skipped - akin to https://stackoverflow.com/a/54540066

One disadvantage of that is that the tests get omitted from the output altogether, rather than being shown as a skipped test. Maybe it's not important though.

Reply:
done